### DISCUSSION TITLE
TCP load balancing

### DISCUSSION THREAD

**Discussion by tuxillo** [linkerd/linkerd2#12492] [General] [answered]
Hi!

According to the documentation the TCP load balancing will be connection based, I assume it's kind of a least connection algo or something similar. Is there any way to influence it besides just TCP connections?

Thanks.

**Comment by adleong** [comment]
Hi @tuxillo, Linkerd uses a load balancing algorithm called EWMA which takes connect latency into account to do weighted load balancing of connections.

**Comment by tuxillo** [comment]
Thanks @adleong but besides that there is really no way to influence it, right?

**Comment by adleong** [comment, marked as answer]
That's right.
